Movecraft/src/main/java/net/countercraft/movecraft/listener/PlayerListener.java

```diff
@@ -102,12 +99,13 @@ public void onPlayerLogout(PlayerQuitEvent e) {
     }
 
     @EventHandler
-    public void onPlayerDeath(EntityDamageByEntityEvent e) {
+    public void onPlayerDeath(PlayerDeathEvent e) {
         // changed to death so when you shoot up an airship and hit the pilot, it still sinks
-        if (!(e instanceof Player))
+
+        if (!Settings.ReleaseOnDeath)
             return;
 
-        Player p = (Player) e;
+        Player p = e.getPlayer();
```

Comment:
I'm not sure if this is the intent, but it only releases the craft and does not sink it.  Based on the comment I think it should sink and not release.

Reply:
Not sure what would be the best behaviour in this case, releasing the craft or sink it?

Reply:
Based on the comment, I think sinking is best.  It also will mean that ships will sink, and not be left in air when the player dies.